README.md

```diff
@@ -119,6 +127,11 @@ If you are perfectly content with losing metadata and want to revert to the prev
 
 - [clojure.java.jdbc](https://github.com/clojure/java.jdbc) >= 0.6.0
 - [amazonica](https://github.com/mcohen01/amazonica)
+- [carmine](https://github.com/ptaoussanis/carmine)
+
+## Development
+
+Start databases with a command `docker-compose up` before running the tests
```

Comment:
I need to understand this better...Are you saying that I need two commands to run the unit-tests (`docker-compose up` followed by `lein test`)? If yes, could we not automate this via leiningen?

Reply:
I'm very open to the idea of using containerised DBs for the purposes of testing. When I said I need to understand this better, I basically meant what tools do I need (e.g `docker`, `docker-compose`), and more importantly how we can make integrate this step (the extra command) with leiningen's profiles. I'm on ubuntu, so  I will look into installing docker this evening.

Reply:
Great! I updated the instructions a bit now. Since `docker-compose` is a shell command I don't know how easy it is to integrate with leiningen. There is a plugin for Leiningen (https://github.com/hypirion/lein-shell) for running shell commands but I am not sure if it makes sense to take that extra dependency. What do you think?